Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue3548.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Xamarin.Forms.Internals;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Controls.Effects;
+using System.Threading.Tasks;
+
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 3548, "Cannot attach effect to Frame", PlatformAffected.Android, NavigationBehavior.PushAsync)]
+	public class Issue3548 : ContentPage
```

Comment:
@AndreiMisiukevich can you automate this test?

Reply:
I added something. But not sure if it works indeed. Because i cannot run UITests on my mac. It always shows failure message